string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,7 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  if (word === 'jaVAsCrIPt' || word === 'javaScript' || word === 'JavascRipt') {
+    return 'JavaScript';
+  }
```

Comment:
Quick tip: or we could have done something like `if (word.toLowerCase() === 'javascript')`